nmdc_server/config.py

```diff
@@ -36,7 +36,8 @@ class Settings(BaseSettings):
     host: Optional[str] = None  # sets the host name for the oauth2 redirect
 
     # mongo database used for ingest
-    mongo_host: str = "nmdc-metadata.polyneme.xyz"
+    mongo_host: str = "mongo-loadbalancer.nmdc-runtime-dev.development.svc.spin.nersc.org"
+    mongo_port: int = 27017
```

Comment:
Do any of our environments use a nonstandard mongo port? This is a good chance, I was just curious.

Another option we might want to consider for the future is to just accept a URI connection string that can encode lots of different options, rather than enumerating the options explicitly.

Reply:
Sorta.  You have to do SSH tunneling now to run this locally.

And that command doesn't actually work, I had to modify it a bit, which I'll document elsehwhere.

https://github.com/microbiomedata/nmdc-runtime/blob/44fbb11f69b51866a410a463f1f5678a2391cd8b/Makefile#L65